added common scenarios (#71075)
Azure/azure-content: articles/active-directory-b2c/add-password-reset-policy.md

![Password reset flow](./media/add-password-reset-policy/password-reset-flow.png)

> Note: Password reset flow is only applicable to local accounts in Azure AD B2C with [email address](https://docs.microsoft.com/azure/active-directory-b2c/identity-provider-local?pivots=b2c-user-flow#email-sign-in) or [username](https://docs.microsoft.com/azure/active-directory-b2c/identity-provider-local?pivots=b2c-user-flow#username-sign-in) with password and does not apply to federated accounts. 

Reviewer: ```suggestion
Note: The password reset flow is only applicable to local accounts in Azure AD B2C that use an [email address](https://docs.microsoft.com/azure/active-directory-b2c/identity-provider-local?pivots=b2c-user-flow#email-sign-in) or [username](https://docs.microsoft.com/azure/active-directory-b2c/identity-provider-local?pivots=b2c-user-flow#username-sign-in) with a password for sign-in. The password reset flow doesn't apply to federated accounts. 
```

::: zone-end

::: zone pivot="b2c-custom-policy"

## Common password reset scenarios 

### Force password reset first logon

> A common practice after migrating users to Azure AD B2C with random passwords, is to have the users verify their email address and reset their password during first login. Please find the sample [here](https://github.com/azure-ad-b2c/samples/tree/master/policies/force-password-reset-first-logon) for this scenario. 

Reviewer: ```suggestion
A common practice after migrating users to Azure AD B2C with random passwords is to have the users verify their email addresses and reset their passwords during first sign-in. [See our code sample](https://github.com/azure-ad-b2c/samples/tree/master/policies/force-password-reset-first-logon) for this scenario. 
```

### Password reset without being able to use the last password

> Some security practices demands that the new password created is not the same as the old password. Please find the sample [here](https://github.com/azure-ad-b2c/samples/tree/master/policies/password-reset-not-last-password) for this scenario. 

Reviewer: ```suggestion
Some security practices require the newly created password to be different from the old password. [See our code sample](https://github.com/azure-ad-b2c/samples/tree/master/policies/password-reset-not-last-password) for this scenario. 
```
